32blit-pico/audio.cpp

```diff
@@ -42,24 +71,84 @@ void init_audio() {
   bool ok = audio_i2s_connect(producer_pool);
   assert(ok);
   audio_i2s_set_enabled(true);
+#endif
+
+#ifdef AUDIO_PWM
+  struct audio_pwm_channel_config audio_pwm_config = {
+    .core = {
+      .base_pin = PICO_AUDIO_PWM_MONO_PIN,
+      .dma_channel = 1,
+      .pio_sm = 1,
+    },
+    .pattern = 3,
+  };
+  output_format = audio_pwm_setup(&audio_format, -1, &audio_pwm_config);
+  if (!output_format) {
+      panic("PicoAudio: Unable to open audio device.\n");
+  }
+#if OVERCLOCK_250
```

Comment:
Think this'll need `#include "config.h"`

Reply:
Oof, done! Thank you.